frontend/post/post_details.html

```diff
@@ -171,7 +171,9 @@ <h3 md-colors="{color:'teal-800'}">{{postDetailsCtrl.post.shared_event.start_tim
   <div class="youtube-player" ng-if="postDetailsCtrl.showVideo(postDetailsCtrl.post)">
     <iframe ng-src="{{postDetailsCtrl.getVideoUrl(postDetailsCtrl.post)}}"></iframe>
   </div>
-  <pdf-view pdf-files='postDetailsCtrl.post.pdf_files' is-editing="false"></pdf-view>
+  <pdf-view pdf-files='postDetailsCtrl.post.pdf_files' is-editing="false"
+    ng-hide="postDetailsCtrl.isDeleted(postDetailsCtrl.post)">
```

Comment:
por que ng-hide e não ng-if?

Reply:
Eu usei o ng-hide por ser mais rápido de processar no html e por melhorar a semântica do código, pois eu quero que esconda quando o post for deletado
hide <- isDeleted é mais fácil de entender do que if <- ! isDeleted